async-request-reply/README.md

```diff
@@ -84,21 +111,58 @@ For more information about this pattern, see [Asynchronous Request-Reply pattern
 1. Delete all the Azure resources for this Cloud Async Pattern
 
    ```bash
-   az group delete -n rg-asyncrequestreply -y
+   az group delete -n ${RESOURCEGROUP} -y
    ```
 
 ### Running localy
 
 You could open the solution with Visual Studio, then you need to create on the root `local.settings.json`
 
-   ```json
-    {
-    "IsEncrypted": false,
-    "Values": {
-        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-        "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-        "ServiceBusConnectionAppSetting": "<yourdata>",
-        "StorageConnectionAppSetting": "<yourData>"
-    }
-    }
-   ```
\ No newline at end of file
+```json
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
+    "ServiceBusConnection__fullyQualifiedNamespace": "<yourData>",
+    "DataStorage__blobServiceUri": "<yourData>"
+  }
+}
+```
+
+As far the implementation is using manage identity, you need to assign the role to your [developer identity](https://learn.microsoft.com/azure/azure-functions/functions-reference?tabs=blob&pivots=programming-language-csharp#local-development-with-identity-based-connections).
```

Comment:
What are they supposed to DO here?  Issue a command?  Put this bicep in a file?  Do something from the portal?

Reply:
good catch. We need to include this in the bicep file.  I will fix it.

Reply:
done